### ISSUE TITLE
Support for 2FA

### ISSUE THREAD

**Issue by Mondei1** [ch-a-os/DocSort#40]
Because we offering the ability to host your own DocSort server we need some security layers and one of the best options is 2FA (Two factor authentification). It's simple to setup, doesn't need any technical knowledge of the user and is fast to use at login. As developer it's easy to implement too (as example [this](https://www.npmjs.com/package/2fa) library) and can be a great security layer. 

I'll watch out for more security layers that can be added (like awnser some questions) to reach maximum security. I think for a program like DocSort this is extremly necessary to protect your documents for others.

**Comment by 720degreeLotus** [comment]
You are right, but i maybe would not link it to the v1.0 milestone... Not sure how this develops but since this is just an enhancement of security, this maybe (not 100% sure) belongs more after the 1.0-milestone...

**Comment by Mondei1** [comment]
I created a new milestone called "v.1.x-NEXT" and putted this issue in it.
